client/src/pages/Edit.js

```diff
@@ -62,44 +64,46 @@ function Edit() {
   };
   return (
     <>
+      <Layout />
       <Container>
-        <h1>Edit Food Items!</h1>
         <div>
           <form onSubmit={FormHandle}>
             <div>
-              <h2>{data.data.title}</h2>
-              <label>Food Title</label>
+              <label>Title</label>
               <input
                 type="text"
                 name="title"
                 value={editFood.title}
                 onChange={onInputChange}
               />
+              <h2>{data.title}</h2>
```

Comment:
This feels off to me.  I thought the goal was to set the value of the input to `data.title`.  Where did I lead you astray?